doc/user/content/manage/blue-green.md

```diff
@@ -100,26 +100,28 @@ as “caught up”, and you can compare both the `prod` and `prod_deploy` versio
 by viewing the Workflow graph from a common source or other upstream
 Materialization.
 
-1. Perform your end-to-end application tests on `prod_deploy` objects to ensure
+2. Perform your end-to-end application tests on `prod_deploy` objects to ensure
 it is safe to cut over.
 
-1. Use the `SWAP` operation to atomically rename your objects in a way that is
+3. Use the `SWAP` operation to atomically rename your objects in a way that is
 transparent to clients.
 
-  ```sql
-  BEGIN;
-  ALTER SCHEMA prod SWAP WITH prod_deploy;
-  ALTER CLUSTER prod SWAP WITH prod_deploy;
-  COMMIT;
-  ```
+    ```sql
+    BEGIN;
+    ALTER SCHEMA prod SWAP WITH prod_deploy;
+    ALTER CLUSTER prod SWAP WITH prod_deploy;
+    COMMIT;
+    ```
+
+    {{< note >}}This transaction may fail to commit if the objects are concurrently modified by a separate session. If this occurs you should re-run the transaction.{{</ note >}}
```

Comment:
At least in https://github.com/MaterializeInc/materialize/issues/24196 this error was not happening in the swap, but in SELECTs which were running during the swap.